openfl/federated/__init__.py

```diff
@@ -3,14 +3,15 @@
 
 """openfl.federated package."""
 
-import pkgutil
+import importlib.util
 from .plan import Plan  # NOQA
 from .task import TaskRunner  # NOQA
 from .data import DataLoader  # NOQA
 
-if pkgutil.find_loader('tensorflow'):
-    from .task import TensorFlowTaskRunner, KerasTaskRunner, FederatedModel  # NOQA
+if importlib.util.find_spec('tensorflow'):
```

Comment:
That would look cleaner. Let's address in a separate PR, though. I actually only updated this because a warning kept popping up for me that pkgutil was being discontinued. Could even discard this commit if we don't want it